tuna/corrupt_configs.py

```diff
@@ -47,31 +47,50 @@ def get_logger():
   return logger
 
 
-def main():
+LOGGER = get_logger()
+
+
+# pylint: disable-msg=too-many-locals
+def main() -> None:
   """Main module function"""
-  file_name = None
-  arch = None
+  file_name: Optional[str] = None
+  arch: Optional[str] = None
+
   if len(sys.argv) == 3:
     file_name = sys.argv[1]
     arch = sys.argv[2]
   else:
     sys.exit("Usage: " + sys.argv[0] + " <output_file> <arch>")
 
-  table_cols_conv_invmap = {v: k for k, v in TABLE_COLS_CONV_MAP.items()}
-  table_cols_fusion_invmap = {v: k for k, v in TABLE_COLS_FUSION_MAP.items()}
+  idx_1: Dict[Any, Any]
+  idx_2: Dict[Any, Any]
+  table_cols_conv_invmap: dict = {
+      idx_1: idx_2 for idx_2, idx_1 in TABLE_COLS_CONV_MAP.items()
+  }
+  table_cols_fusion_invmap: dict = {
+      idx_1: idx_2 for idx_2, idx_1 in TABLE_COLS_FUSION_MAP.items()
+  }
```

Comment:
TABLE_COLS_CONV_MAP is a dict of tuples.
So calling items on this you would get (key, tuple)
here idx_2 = key
and idx_1 = tuple

Reply:
idx_1  to key_idx and idx_2 to tuple_idx , more readability